apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.Snippets/ErrorReportingSnippets.cs

```diff
@@ -0,0 +1,186 @@
+﻿// Copyright 2018 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.ClientTesting;
+using Google.Cloud.Diagnostics.Common.IntegrationTests;
+using Google.Cloud.Diagnostics.Common.Tests;
+using Google.Cloud.ErrorReporting.V1Beta1;
+using Microsoft.Owin.Testing;
+using Owin;
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+using Xunit;
+
+namespace Google.Cloud.Diagnostics.AspNet.Snippets
+{
+    [SnippetOutputCollector]
+    public class ErrorReportingSnippets
```

Comment:
None of this looks like it's going to be suitable for snippets - it looks like it really is testing the logger.
What's the reason behind moving these tests out of the regular integration tests?

Reply:
Well I was following advice you gave me for BigQuery here.

Reply:
I suspect that was for a situation where the tests *were* snippets :) Happy to have a VC if that would help clarify.

Reply:
Hmmm, ok, I was describing that same difference in my comment.